docs/05-engineering/security-compliance.md

```diff
@@ -2,41 +2,28 @@
 
 ## Security Policy
 
-We understand and abide by the [CivicActions Employee/Contractor Security Policy](../03-policies/security.md). In particular:
+All engineers understand and abide by the [CivicActions Employee/Contractor Security Policy](../03-policies/security.md). Further, we have taken care in following all the steps laid out in the [Security Training](../01-welcome-to-civicactions/training/security-training.md).
 
-* We practice [Server & Site Security](../03-policies/security.md#server--site-security)
-* using only sanitized databases
-* taking care to not install restricted access files on development or personal instances outsite the project defined security boundary
-* and scrubbing unneeded data from our development systems
-
-## Awareness and Tools
-
-We enhance our [Security Awareness with Tools](../09-security/awareness.md) by:
-
-* [Securing our Laptops](../09-security/awareness.md#securing-your-laptop)
-* [Using Password Management Tools](../09-security/awareness.md#password-management-tools)
-* [Using Multi-Factor Authentication](../09-security/awareness.md#tfa)
-* [Increasing our awareness of Phishing and Social Engineering](../09-security/awareness.md#phishing-and-social-engineering)
-* [Keeping our Personal Systems up-to-date](../09-security/awareness.md#keep-your-systems-up-to-date)
-* [Employing Disk Encryption and Secure Storage Management](../09-security/awareness.md#disk-encryption-and-storage-management)
+In particular:
 
-## Training
-
-* We perform a yearly review of the [CivicActions Employee/Contractor SecurityPolicy](../03-policies/security.md)
-* We train all our employees and contractors in PII Awareness
+* We practice [Server & Site Security](../03-policies/security.md#server--site-security)
+  * using only sanitized databases
+  * taking care to not install restricted access files on development or personal instances outsite the project defined security boundary
+  * and scrubbing unneeded data from our development systems
 
 ## As Developers
 
 * We minimize custom code, always preferring to use community maintained modules and contribute patches when needed
 * When necessary for new functionality, we strive to create generic modules and contribute them to the parent project
 * Custom code must:
-* have an associated Jira (or other ticketing system) ticket
-* include testing mechanisms, ideally hooked into the continuous integration pipeline
-* conform to coding standards (use static code analysis where possible (such as [DCQ](https://www.drupal.org/project/dcq))
-* undergo security peer review
+  * have an associated Jira (or other ticketing system) ticket
```

Comment:
@openprivacy unfortunately, Read The Docs enforces a strict markdown spec that requires four spaces (which is fugly and annoying) for nested lists. For now, can you change all you nested lists to four spaces?